lib/types/types.ts

```diff
@@ -61,7 +61,7 @@ export interface FieldDefinition {
 export interface ParquetField {
     name: string;
     path: string[];
-    key: string;
+    // key: string;
```

Comment:
It is a required field here but the code does not include keys when making `ParquetField` types